CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### Unreleased
   - Ensure integrity and version of library dependencies
   - Integrate with sdc-rabbit library
+  - Stop rabbit consumer checking for tx_id
```

Comment:
This should probably be something like:

Pass `tx_id=False` to process method to not check if a tx_id is received